src/zfs_arc.c

```diff
@@ -285,10 +285,20 @@ static int za_read(void) {
   za_read_gauge(ksp, "mfu_size", "cache_size", "mfu_size");
   za_read_gauge(ksp, "mru_ghost_size", "cache_size", "mru_ghost_size");
   za_read_gauge(ksp, "mru_size", "cache_size", "mru_size");
-  za_read_gauge(ksp, "other_size", "cache_size", "other_size");
   za_read_gauge(ksp, "p", "cache_size", "p");
   za_read_gauge(ksp, "size", "cache_size", "arc");
 
+  /* The "other_size" value was replaced by more specific values in ZFS on Linux
+   * version 0.7.0 (commit 25458cb)
+   */
+  static int other_size_avail = 1;
```

Comment:
can you make this a bool?

Reply:
or even better, drop the use of the static variable, and just read other_size if you can't read one of the other 3 counters,

Reply:
I was mimicking the logic from the "l2_size_avail" right below this, because I assumed that to be the desired approach. But sure, here you go.